### PR TITLE
release: v1.4.4

### PR DESCRIPTION
**Pull request by ttaylorr**
cc @technoweenie @rubyist @sschuberth

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+## 1.4.4 (21 October, 2016)
+
+### Bugs
+
+* commands,lfs/tq: Only send unique OIDs to the Transfer Queue #1600 (@ttaylorr)
+
+### Misc
+
+* Expose the result message in case of an SSH authentication error #1599
+* (@sschuberth)
```
Comment (technoweenie) [inline CHANGELOG.md]: I'd move this under Bugs. The Misc section is for project related stuff that doesn't affect how git-lfs works, like the appveyor change.

Comment (ttaylorr) [inline CHANGELOG.md]: Good call, I updated to address.

**Comment by ttaylorr** [comment]
Planning on taking a look at a few more issues before I cut this release on Monday.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Git LFS Changelog
 
+## 1.4.4 (21 October, 2016)
+
+### Bugs
+
+* commands,lfs/tq: Only send unique OIDs to the Transfer Queue #1600 (@ttaylorr)
```
Comment (technoweenie) [inline CHANGELOG.md]: I think the package prefix is unnecessary for the changelog, as it's only relevant to lfs developers, and not end users. Up to you whether that warrants a change or not though.

**Comment by ttaylorr** [comment]
Sorry for the force push, I wanted to merge this in as a single commit without squashing.

**Pull request merged**